great_expectations_cloud/agent/actions/run_scheduled_window_checkpoint.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from typing_extensions import override
+
+from great_expectations_cloud.agent.actions.agent_action import (
+    ActionResult,
+    AgentAction,
+)
+from great_expectations_cloud.agent.actions.run_window_checkpoint import run_window_checkpoint
+from great_expectations_cloud.agent.event_handler import register_event_action
+from great_expectations_cloud.agent.models import (
+    RunScheduledWindowCheckpointEvent,
+)
+
+
+class RunScheduledWindowCheckpointAction(AgentAction[RunScheduledWindowCheckpointEvent]):
```

Comment:
@rreinoldsc Do we need to add this to one of the `__init__.py` files like we did for the `RunWindowCheckpointAction`?

(see `agent/actions/__init__.py`)

Reply:
yes ty, added in 5ce69d9